code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -61,7 +61,7 @@
 	clothes_req = 0
 	range = 5
 	action_icon_state = "veil"
-	var/blacklisted_lights = list(/obj/item/device/flashlight/flare, /obj/item/device/flashlight/slime)
+	var/blacklisted_lights = list(/obj/item/device/flashlight/slime)//Slightly buffed, requiring slimes instead of slimes and flares
```

Comment:
Keep in mind that there is now a second part to each shadowling round, the buffs were created with this in mind.

Ideally as soon as shadowlings are screamed over the radio (and assuming botany was assimilate), assistants/the warden should already be planting glowshrooms everywhere.